src/Sentry.Unity.Editor/Il2CppOption.cs

```diff
@@ -0,0 +1,24 @@
+using UnityEditor;
+using UnityEditor.Build;
+using UnityEditor.Build.Reporting;
+using UnityEngine;
+
+namespace Sentry.Unity.Editor
+{
+    class Il2CppOption : IPreprocessBuildWithReport
+    {
+        public int callbackOrder => 0;
+
+        public void OnPreprocessBuild(BuildReport report)
+        {
+            if (!Application.unityVersion.StartsWith("2022.1"))
```

Comment:
We won't want this also on 2022.2, 2022.3, etc?

```suggestion
            if (!Application.unityVersion.StartsWith("2022"))
```

Maybe a comment above to when do we want or not this thing to happen?